app/assets/javascripts/sitewide/file_preview.js

```diff
@@ -1,13 +1,16 @@
 /* eslint no-underscore-dangle: ["error", { "allowAfterThis": true }]*/
 /* eslint no-use-before-define: ["error", { "functions": false }]*/
 /* eslint-disable no-underscore-dangle */
-/* global Uint8Array fabric tui animateSpinner setupAssetsLoading I18n PerfectScrollbar*/
+/* global Uint8Array fabric tui animateSpinner Assets I18n PerfectScrollbar*/
 //= require assets
 
 var FilePreviewModal = (function() {
   'use strict';
 
   var readOnly = false;
+  var CHECK_READY_DELAY = 5000;
+  var CHECK_READY_TRIES_LIMIT = 60;
```

Comment:
Maybe we can move this constants to one place. I create constant file here - https://github.com/biosistemika/scinote-web/pull/1800/files#diff-b568eac5c36c99e4e1dd3c39ae028462